cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -270,6 +271,10 @@ private static void warningsForRemovedEndVars(Map<String, String> map) {
             LOGGER.warn("Cruise Control TLS sidecar container has been removed and the environment variable {} is not used anymore. " +
                     "You can remove it from the Strimzi Cluster Operator deployment.", STRIMZI_DEFAULT_TLS_SIDECAR_CRUISE_CONTROL_IMAGE);
         }
+        if (map.containsKey(STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE))    {
+            LOGGER.warn("Entity Operator TLS sidecar container has been removed and the environment variable {} is not used anymore. " +
```

Comment:
"has been removed" or "is no longer used"?

Reply:
Well, both. The container "has been removed", and the configuration "is no longer used". Keeping the current version for consistency with the other warning messages.